ODIN_II/regression_test/benchmark/suite/temp_suite/task_list.conf

```diff
@@ -0,0 +1 @@
+regression_test/benchmark/task/keywords/*
```

Comment:
rename the suite to maybe keyword?

Reply:
My plan was to delete the suite, I only added it for generating results but should I keep it?

Reply:
I think its better that way yeah, it is a suite after all!